Nav branch

MrkarlosM opened the pull request:
This pull request is for code reviewing.

MrkarlosM committed:
Starting with NavBar

Added Elements and Some Styling.

AhmedAmin90 committed:
finish navbar and start on main part

Finished navbar with style css, and made first elements in HTML for the main part

MrkarlosM committed:
Add Style to Image

Styling Image and adding text description.

AhmedAmin90 committed:
Added icons and started on paragraph

Added icons under image and styled them.
Added first part of the paragraph

MrkarlosM committed:
Added Second Paragraph

Added second paragraph and styling for it.

AntonyOtero commented on index.html:
```diff
@@ -8,22 +8,48 @@
+    <main>
+        <div class="container">
+            <div class="feature-img-post">
+                <h1>Space Ripples Reveal Big Bang’s <br> Smoking Gun</h1>
```
Your main header could be made bold and centered to better match the article's title.

abongsjoel commented on src/css/style.css:
```diff
@@ -3,30 +3,57 @@
+nav i{
+  padding: 8px 9px 9px;
+  font-size: 15px;
+  color: #333333;
```
You can use the short form of the hex color values here as in 
```
color: #333;
```

AhmedAmin90 committed:
Added inflation theory images with grid

we used grid model to build this part.

MrkarlosM committed:
Added paragraphs and video

Added some paragraphs, style them, added video, and second main image.

AhmedAmin90 committed:
added some images

screen shots

AhmedAmin90 committed:
added last part of the paragraph - comment section

MrkarlosM committed:
Added grid modeling

Added color to grid's borders.

AhmedAmin90 committed:
started on more articles part

AhmedAmin90 committed:
comment"trial commit"

MrkarlosM committed:
Finishing most popular and aside part

Added mot popular links, added aside part of page.

AhmedAmin90 committed:
started on footer part

added main sections and started on them

MrkarlosM committed:
Final part

Did final list in the rightmost part and added icons.

MrkarlosM committed:
Finishing styles

Added Style to main title.
Added pictures.
Added alts to pictures.
Added styling on last texts.

MrkarlosM committed:
Create linters.yml

MrkarlosM committed:
Added last files to push

Added Readme and License, also organizing files and adding comments.
